students/BrianNisonger/session4/trigram.py

```diff
@@ -0,0 +1,59 @@
+"""
+trigrams
+"""
+"""
+set default
+"""
+def make_words(source):
+    words=""
+    with open(source) as f:
+        lines=f.readlines()
+    punct=[",",".","-","(",")","\n","!","'s","'","?","_",";"]   
+    for line in lines:
+        for punc in punct:
+            line=line.replace(punc," ")
+        words=words+line
```

Comment:
do you get a space between the lines?

But anyway -- it is inefficient to build up a long string by adding pieces one at a time. Better to process each line into a list of words, and accumulate that in a list -- you want a list in the end anyway.

-CHB